spec/spec_helper.rb

```diff
@@ -1,16 +1,14 @@
-ENV['RACK_ENV'] = "test"
+ENV['RACK_ENV'] = 'test'
+
+Dir['./spec/support/**/*.rb'].sort.each { |f| require f}
 
-require 'rubygems'
-require 'bundler/setup'
 require 'mongoid'
```

Comment:
the only thing i can think of is that we don't have a requirement for mongoid in gemspec but you need it in order to run the tests.